platform/app/src/appInit.js

```diff
@@ -40,7 +40,7 @@ async function appInit(appConfigOrFunc, defaultExtensions, defaultModes) {
 
   const appConfig = {
     ...(typeof appConfigOrFunc === 'function'
-      ? await appConfigOrFunc({ servicesManager })
+      ? await appConfigOrFunc({ servicesManager, loadModules })
```

Comment:
The load modules is added here so that the app config can use modules listed in the loadModules.